imports/collections/schemas/products.js

```diff
@@ -585,6 +585,10 @@ export const Product = new SimpleSchema({
     optional: true,
     defaultValue: {}
   },
+  "currentProductHash": {
+    type: String,
+    optional: true
```

Comment:
I guess I could see it either way, it's just used for comparison purposes, so it gets used with `publishedProductHash`. If that one is optional, they both can be?

Reply:
In my opinion the bias would always be towards making something not optional if something is going to be relied on by another part of the program, except where we know that it won't always be available. Which it seems like this would always be available since it's calculated on the record itself.

Reply:
If we make it required we need to make sure any sample data is generated with a hash as well

Reply:
It's not being automatically created on insert?

Reply:
That makes sense. I’ll need a migration if we make it required, right?

Reply:
I'd think we'd want one for any legacy data. Not sure how the default product is being generated right now, but might look there as well?